lib/zk_promise.js

```diff
@@ -24,11 +24,14 @@ class ZooKeeperPromise extends ZooKeeper {
      * @param {string} path
      * @param {(string|Buffer)} data
      * @param {number} flags
+     * @param {number} ttl,
+     *   Must be positive integer if using any TTL mode;
```

Comment:
I think the data type should be something like number or undefined, otherwise (I think) the type hint will complain about a missing param.